amm/interp/src/main/scala/ammonite/interp/Interpreter.scala

```diff
@@ -195,6 +194,10 @@ class Interpreter(val printer: Printer,
           else headFrame.addClasspath(res.files.map(_.toNIO.toUri.toURL))
 
           Res.Success(Imports())
+
+        case res: ImportHook.Result.Repo =>
+          Res.Success(Imports())
```

Comment:
Wouldn't it work to just call `repositories.update` here, instead of adding `addRepository`?

Reply:
Hah, good point, I'll change it

Reply:
https://github.com/lihaoyi/Ammonite/pull/1090/commits/e532f58e3e73812713a936b91b67b2c34fe3e97a